src/views/compute/DestroyVM.vue

```diff
@@ -136,7 +136,7 @@ export default {
             catchMessage: this.$t('error.fetching.async.job.result'),
             successMessage: `${this.$t('message.success.delete.vm')} ${this.resource.name}`,
             successMethod: () => {
-              if (values.expunge) {
+              if ('id' in this.resource && values.expunge) {
```

Comment:
@utchoang This would return the user back to the dashboard if he tries to expunge the vm via Quickview. Could you add a check to go back onlyif the url !== '/vm'
![b](https://user-images.githubusercontent.com/8244774/94402298-7f21c900-0189-11eb-9ea0-154565c66ff0.jpg)